src/meta/TableSpec.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <regex>
```

Comment:
I may be wrong but I don't think this is used.

Reply:
removed